word_cloud_generator/word_cloud_generator.py

```diff
@@ -0,0 +1,307 @@
+import re
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from pathlib import Path
+from sys import argv
+from time import sleep
+from os import path
+from wordcloud import WordCloud
+from typing import Optional
+from dataclasses import dataclass
+from mashumaro import DataClassDictMixin
+
+import json
+import requests
+
+
+@dataclass
+class IssuesFilter(DataClassDictMixin):
+    output: str
+    start_date: str
+    end_date: str
+    word_cloud: str
+    assigned: Optional[str] = None
+    filter_for: Optional[str] = None
+    filter_not: Optional[str] = None
+
+
+def from_user_inputs(**kwargs):
+    """
+    Take the inputs from a argparse and populate a IssuesFilter dataclass and return it
+    :param kwargs: a dictionary of argparse values
+    """
+
+    return IssuesFilter.from_dict(kwargs)
```

Comment:
Do you need to use `.from_dict()` here?
I'm wondering if this would be better?
```suggestion
    return IssuesFilter(**kwargs)
```